### PR TITLE
Fix xpu error message

### PR DESCRIPTION
**Pull request by MrChengmo**
### PR types

Bug fixes

### PR changes

OPs

### Describe

修复sgd、momentum xpu op的报错信息

### CODE REVIEW

File: paddle/fluid/operators/optimizers/momentum_op_xpu.cc [hunk 1]
```diff
@@ -48,8 +48,24 @@ class MomentumOpXPUKernel : public framework::OpKernel<T> {
...
+    if (r == xpu::Error_t::INVALID_PARAM) {
+      PADDLE_ENFORCE_EQ(
+          r, xpu::Error_t::SUCCESS,
+          platform::errors::InvalidArgument(
+              "XPU kernel error of MomentumOp, error message: INVALID_PARAM, "
+              "please check your input & output."));
+    } else if (r == xpu::Error_t::RUNTIME_ERROR) {
+      PADDLE_ENFORCE_EQ(
+          r, xpu::Error_t::SUCCESS,
+          platform::errors::PermissionDenied(
```
Comment (chenwhql) [inline paddle/fluid/operators/optimizers/momentum_op_xpu.cc]: maybe `Unavailable` is better?

File: paddle/fluid/operators/optimizers/momentum_op_xpu.cc [hunk 2]
```diff
@@ -48,8 +48,24 @@ class MomentumOpXPUKernel : public framework::OpKernel<T> {
...
+              "XPU kernel error of MomentumOp, error message: RUNTIME_ERROR, "
+              "please check whether Baidu Kunlun card is properly installed."));
+    } else if (r == xpu::Error_t::NO_ENOUGH_WORKSPACE) {
+      PADDLE_ENFORCE_EQ(r, xpu::Error_t::SUCCESS,
+                        platform::errors::OutOfRange(
```
Comment (chenwhql) [inline paddle/fluid/operators/optimizers/momentum_op_xpu.cc]: maybe `ResourceExhausted` is better?

File: paddle/fluid/operators/optimizers/sgd_op_xpu.cc [hunk 3]
```diff
@@ -56,9 +56,25 @@ class SGDOpXPUKernel : public framework::OpKernel<T> {
...
-          r, xpu::Error_t::SUCCESS,
-          platform::errors::PermissionDenied("XPU kernel error!"));
+      if (r == xpu::Error_t::INVALID_PARAM) {
+        PADDLE_ENFORCE_EQ(
+            r, xpu::Error_t::SUCCESS,
+            platform::errors::InvalidArgument(
+                "XPU kernel error of SgdOp, error message: INVALID_PARAM, "
+                "please check your input & output."));
+      } else if (r == xpu::Error_t::RUNTIME_ERROR) {
+        PADDLE_ENFORCE_EQ(r, xpu::Error_t::SUCCESS,
```
Comment (chenwhql) [inline paddle/fluid/operators/optimizers/sgd_op_xpu.cc:66]: same above

File: paddle/fluid/operators/optimizers/sgd_op_xpu.cc [hunk 4]
```diff
@@ -56,9 +56,25 @@ class SGDOpXPUKernel : public framework::OpKernel<T> {
...
+                          platform::errors::PermissionDenied(
+                              "XPU kernel error of SgdOp, error message: "
+                              "RUNTIME_ERROR, please check whether Baidu "
+                              "Kunlun Card is properly installed."));
+      } else if (r == xpu::Error_t::NO_ENOUGH_WORKSPACE) {
+        PADDLE_ENFORCE_EQ(
+            r, xpu::Error_t::SUCCESS,
+            platform::errors::OutOfRange("XPU kernel error of SgdOp, error "
```
Comment (chenwhql) [inline paddle/fluid/operators/optimizers/sgd_op_xpu.cc]: same above

File: paddle/fluid/operators/optimizers/momentum_op_xpu.cc (hunk 1 above)
Comment (MrChengmo) [reply paddle/fluid/operators/optimizers/momentum_op_xpu.cc]: fixed

**Review by chenwhql — APPROVED** [review]
LGTM

**Pull request merged**